buildrunner/docker/runner.py

```diff
@@ -397,8 +392,8 @@ def restore_caches(self, logger: ContainerLogger, caches: OrderedDict) -> None:
                         f"{actual_cache_archive_file} at the path {docker_path}\n"
                     )
 
-            except docker.errors.APIError as exception:
-                logger.write(f"WARNING: Encountered exception\n{exception}\n")
+            except docker.errors.APIError:
+                logger.exception("Encountered exception")
```

Comment:
Why wouldn't we want to log the exception message here?

Reply:
`logger.exception` does just that.